source/slang/hlsl.meta.slang

```diff
@@ -11404,19 +11404,64 @@ ${{{{
 }
 
 //
-// Vulkan/SPIR-V specific features
+// SubpassInput
 //
 
-struct VkSubpassInput<T>
+__magic_type(SubpassInputType)
+__intrinsic_type($(kIROp_SubpassInputType))
+struct __SubpassImpl<T, let isMS:int>
 {
-    T SubpassLoad();
 }
 
-struct VkSubpassInputMS<T>
+__generic<T, let isMS:int=0>
+typealias SubpassInputMS = __SubpassImpl<T, isMS>;
+
+__generic<T, let isMS:int=0>
```

Comment:
Default generic argument value in an extension have no meaning.

Reply:
I believe things in this extension can be defined directly in the original `struct` type. If you intend to provide `SubpassLoad` for `isMS = 0` only, you should write:
```
__generic<T>
extension __SubpassImpl<T, 0>
{
    // ...
}
```